cmd/earth/main.go

```diff
@@ -716,6 +728,15 @@ func (app *earthApp) run(ctx context.Context, args []string) int {
 
 func (app *earthApp) actionBootstrap(c *cli.Context) error {
 
+	switch app.homebrewSource {
+	case "bash":
+		fmt.Printf(bashCompleteEntry)
+		os.Exit(0)
```

Comment:
Can we just return `nil` here so that os.Exit is only used in main?

Reply:
that's a good point; for some reason I was worried it would keep running but that's not the case. I'll return nil.